docs/src/views/ThemingExamples.tsx

```diff
@@ -90,20 +83,21 @@ export default () => (
         `import { Button } from '@stardust-ui/react'`,
         ``,
         `const styles = {`,
-        `        color: "coral",`,
-        `        backgroundColor: "coral",`,
-        `        fontSize: "14px",`,
-        `        padding: "0 10px",`,
-        `      }`,
+        `  color: "coral",`,
+        `  backgroundColor: "coral",`,
+        `  fontSize: "14px",`,
+        `  padding: "0 10px",`,
+        `}`,
         `const btnExample = () => `,
-        `    <Button`,
-        `      content="Send email"`,
-        `      icon={{`,
-        `        name: "at",`,
-        `        styles: { color: "brown" }`,
-        `      }}`,
-        `      secondary`,
-        `      styles={styles} />`,
+        `  <Button`,
+        `    content="Send email"`,
+        `    icon={{`,
+        `      name: "at",`,
+        `      styles: { color: "brown" }`,
+        `    }}`,
+        `    secondary`,
+        `    styles={styles} />`,
+        `  />`,
```

Comment:
I aligned code in these examples.

<details>
  <summary>Before</summary>

  ![image](https://user-images.githubusercontent.com/14183168/49512887-f3de9180-f897-11e8-9c2a-42fea6f91d35.png)
</details>
<details>
  <summary>After</summary>

![image](https://user-images.githubusercontent.com/14183168/49512961-2d170180-f898-11e8-8602-c6f8caf0b2ff.png)

</details>

Reply:
👍

Reply:
@layershifter I just merged prettier integration #568 which auto formats CodeSnippets and ExampleSnippets.  It also removes the need to have the `value` string prop altogether by printing the `render` prop return value and prettifying it 👍 

Just fix conflicts here to account for that.

Reply:
Not sure how would this reflect the fact that we want to show the examples as Javascript code, but we are writing the actual rendered code in Typescript... I have something like this in the Integrate Custom Components guide. What is the proposal for addressing this?